shared/bus.go

```diff
@@ -143,3 +155,11 @@ func (m *bus) GetConsensusModule() modules.ConsensusModule {
 func (m *bus) GetTelemetryModule() modules.TelemetryModule {
 	return m.telemetry
 }
+
+func (m *bus) GetConfig() map[string]json.RawMessage {
+	return m.config
```

Comment:
This is going to "leak", i.e. allow modification of the returned config. Is this the intention?

Reply:
Also need to use a non-pointer receiver for the method for a Get method.